### ISSUE TITLE
Buttons are vertically ill-aligned

### ISSUE THREAD

**Issue by memeplex** [platformio/platformio-atom-ide-terminal#542]
![screenshot from 2018-06-01 03-13-29](https://user-images.githubusercontent.com/2845433/40824280-fd12ee7c-6549-11e8-9544-96b955ed683c.png)

**Comment by the-j0k3r** [comment]
This project is no longer developed.

**Comment by memeplex** [comment]
Sad to hear that since it's such a popular one.

**Comment by the-j0k3r** [comment]
Yep read https://github.com/platformio/platformio-atom-ide-terminal/issues/543

**Comment by the-j0k3r** [comment]
@memeplex try the hack in https://github.com/platformio/platformio-atom-ide-terminal/pull/558 until that gets merged.

**Comment by the-j0k3r** [comment]
@ivankravets please consider reviewing and merging #558 its a non trivial change and a good start into closing this ticket.

**Comment by the-j0k3r** [comment]
@ivankravets I think status-bar needs some update to a newer revision not necessarily the latest but something around 1.4.2

https://github.com/platformio/platformio-atom-ide-terminal/blob/a2769c8958055fbf998b581a332acea045478dad/package.json#L51
